.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
> > I did like the recommendation of using conda build inside the test script to catch any issues with the build scripts.
> I disagree with that

I'm ambivalent on that -- conda is a bi tof a split world right now -- most conda recipes are maintained outside the package.

In this case, the recipe is pretty trivial, so not a big deal either way -- but would the recipe reference a source download in the usual way, or build from the source it is embedded in...

If it's a source download, then it seems keeping the recipe build test separate makes sense.

But as long as travis consistently pases teh conda buld test, then either way is fine :-)